pkg/lib/controller-runtime/client/ssa.go

```diff
@@ -90,10 +90,11 @@ type ServerSideApplier struct {
 // plan := &InstallPlan{}
 // plan.SetNamespace("ns")
 // plan.SetName("install-123def")
-// Eventually(c.Apply(plan, func(p *v1alpha1.InstallPlan) error {
-//		p.Spec.Approved = true
-//		return nil
-// })).Should(Succeed())
+//
+//	Eventually(c.Apply(plan, func(p *v1alpha1.InstallPlan) error {
+//			p.Spec.Approved = true
+//			return nil
+//	})).Should(Succeed())
```

Comment:
Might want to make the whole code block indented. See [Go Doc Comments: Code blocks](https://tip.golang.org/doc/comment#code).

```go
//	plan := &InstallPlan{}
//	plan.SetNamespace("ns")
//	plan.SetName("install-123def")
//	Eventually(c.Apply(plan, func(p *v1alpha1.InstallPlan) error {
//		p.Spec.Approved = true
//		return nil
//	})).Should(Succeed())
```

Reply:
I know. `go fmt` makes assumptions and changes stuff, but not intelegent and breaks code blocks sometimes.

It will be happy with the formatting I suggested.